src/dotnet-interactive.Tests/StdIoBehaviorTests.cs

```diff
@@ -12,6 +15,106 @@
 
 public class StdIoBehaviorTests
 {
+    [Fact]
+    public async Task Pass_Culture_to_stdio_process()
```

Comment:
Let's chat about how to make this test a little simpler.

Reply:
done